api/src/common/decorators_api_views.py

```diff
@@ -0,0 +1,17 @@
+from django.utils.decorators import method_decorator
+from drf_yasg.utils import swagger_auto_schema
+
+# COMMON 6 DECORATORS APPLIED TO ALL MODEL VIEW SET CLASSES
```

Comment:
Just the first letter needs to be capitalized, sorry if I confused you before 😅 . Also, we should talk about the decorators being specifically for swagger